pkg/virt-handler/vm_test.go

```diff
@@ -2596,10 +2597,89 @@ var _ = Describe("VirtualMachineInstance", func() {
 			condition, isBlockMigration := controller.calculateLiveMigrationCondition(vmi)
 			Expect(isBlockMigration).To(BeFalse())
 			Expect(condition.Type).To(Equal(v1.VirtualMachineInstanceIsMigratable))
-			Expect(condition.Status).To(Equal(k8sv1.ConditionFalse))
-			Expect(condition.Reason).To(Equal(v1.VirtualMachineInstanceReasonVirtIOFSNotMigratable))
+			Expect(condition.Status).To(Equal(k8sv1.ConditionTrue))
 		})
 
+		fsPvcVolume := v1.VolumeSource{
+			PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+				PersistentVolumeClaimVolumeSource: k8sv1.PersistentVolumeClaimVolumeSource{
+					ClaimName: "testpvc",
+				},
+			},
+		}
+
+		fsDvVolume := v1.VolumeSource{
+			DataVolume: &v1.DataVolumeSource{
+				Name: "testDV",
+			},
+		}
+
+		errorMsg := "cannot migrate VMI: PVC %s is not shared, live migration requires that all PVCs must be shared (using ReadWriteMany access mode"
```

Comment:
Can you put these and the test cases that use them into a `Context` to show that they belong together?

Reply:
Also, `errorMsg` could be a const.